templates/trader.yaml

```diff
@@ -3,6 +3,7 @@ description: "A single-agent service (sovereign agent) placing bets on Omen"
 hash: bafybeihrogxx2kgl7euvhqsbh33fi4tzuupxodoheraqxd6lkvtvljewtu
 image: https://operate.olas.network/_next/image?url=%2Fimages%2Fprediction-agent.png&w=3840&q=75
 configuration:
+  trader_version: v0.16.4
```

Comment:
This needs to be updated when a new version of trader is meant to be used.

Reply:
We reference it by the hash (which is what is fetched ultimately). What impact does this version have?

Reply:
In particular, we have a recent hash update #225 , but there is no release of the repository. What happens in this situation?

Reply:
@jmoreira-valory this specifies the aea binary that needs to be used when running the agent. It may be that deps change.